python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -64,51 +64,183 @@ def __getitem__(self, key):
         g._meta = g._meta[key]
         return g
 
+    @_dask_cudf_nvtx_annotate
+    def count(self, split_every=None, split_out=1):
+        if _groupby_supported(self):
+            return groupby_agg(
+                self.obj,
+                self.by,
+                {c: "count" for c in self.obj.columns if c not in self.by},
+                split_every=split_every,
+                split_out=split_out,
+                **self.dropna,
```

Comment:
I realize this is allowed, but using a `**` in the middle of the kwargs like this "looks" a bit funny to me. I don't feel stronly about it, but I'd suggest moving these to the end